### PR TITLE
Upgrade Svelte to v5 alpha

### PR DESCRIPTION
**Pull request by trueadm**
This PR updates Svelte to the recent alpha of Svelte 5. It also updates the implementation to make use of the new features available in Svelte 5.

### CODE REVIEW

File: frameworks/keyed/svelte/src/Main.svelte [hunk 1]
```diff
@@ -1,194 +1,186 @@
...
+			data = buildData(1000);
+		},
+		runLots = () => {
+			data = buildData(10000);
+		},
+		swapRows = () => {
+			if (data.length > 998) {
+				const clone = data.slice();
+				const tmp = clone[1];
+				clone[1] = data[998];
```
Comment (gtm-nayan) [inline frameworks/keyed/svelte/src/Main.svelte:92]:
```suggestion
				clone[1] = clone[998];
```

File: frameworks/keyed/svelte/src/Main.svelte [hunk 2]
```diff
@@ -1,194 +1,186 @@
...
-          >
-        </div>
-      </div>
-    </div>
-  </div>
+<div id="main" class="container">
+	<div class="jumbotron">
+		<div class="row">
+			<div class="col-md-6">
+				<h1>Svelte Octane (keyed)</h1>
```
Comment (gtm-nayan) [inline frameworks/keyed/svelte/src/Main.svelte:122]:
```suggestion
				<h1>Svelte v5 Alpha(keyed)</h1>
```

**Comment by birkskyum** [comment]
maybe the non-keyed implementation can be updated as part of this pr as well.

**Comment by krausest** [comment]
npm install failed with the following message:
```
npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: js-framework-benchmark-svelte@1.0.0
npm ERR! Found: svelte@5.0.0-next.1
npm ERR! node_modules/svelte
npm ERR!   dev svelte@"^5.0.0-next.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@">=3.5.0" from rollup-plugin-svelte@7.1.6
npm ERR! node_modules/rollup-plugin-svelte
npm ERR!   dev rollup-plugin-svelte@"7.1.6" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /home/stefan/.npm/_logs/2023-11-12T15_51_50_850Z-eresolve-report.txt
```
I added a local .npmrc with legacy-peer-deps=true such that I can merge the PR. You might want to check that later.

**Comment by krausest** [comment]
Thanks - very exciting:
Screenshot 2023-11-12 at 7 22 44 PM

**Pull request merged**